src/components/nudge/Nudge.scss

```diff
@@ -9,7 +9,7 @@
     padding: 25px;
     background-color: $white;
     border: solid 1px $bdl-gray-02;
-    border-radius: 30px;
+    border-radius: 4px;
```

Comment:
https://github.com/box/box-ui-elements/blob/master/src/styles/constants/_layout.scss#L9

Reply:
thanks, resolved in latest commit